gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -2147,8 +2124,8 @@ filter_supported_accelerators (const gchar ** supported_accelerators)
   idx = 0;
   num_hw = 0;
   while (supported_accelerators[idx] != NULL) {
-    if (get_accl_hw_type (supported_accelerators[idx]) == ACCL_CPU_NEON &&
-        !neon_support) {
+    if (g_strcmp0 (supported_accelerators[idx], ACCL_CPU_NEON_STR) == 0 &&
```

Comment:
g_ascii_strcasecmp() to ignore lowercase?

Reply:
replaced with `g_ascii_strncasecmp()`.